docs/guides/basic-database.rst

```diff
@@ -1,194 +1,106 @@
+Now create the database (on RPM systems, the schema path is ``/usr/share/doc/pdns-backend-sqlite/schema.sqlite3.sql``):
 
+.. code-block:: shell
 
-.. literalinclude:: ../../modules/gmysqlbackend/schema.mysql.sql
-   :language: SQL
+    $ sudo mkdir /var/lib/powerdns
+    $ sudo sqlite3 /var/lib/powerdns/pdns.sqlite3 < /usr/share/doc/pdns-backend-sqlite3/schema.sqlite3.sql
+    $ sudo chown -R pdns:pdns /var/lib/powerdns
 
-We recommend you add the following MySQL statements as well. These will add
-foreign key constraints to the tables in order to automate deletion of records, key
-material, and other information upon deletion of a domain from the
-domains table. These will only work on the InnoDB storage engine, but if you
-followed our guide so far, that's exactly the engine we are using.
+And start PowerDNS
 
-The following SQL does the job:
+.. code-block:: shell
 
-.. literalinclude:: ../../modules/gmysqlbackend/enable-foreign-keys.mysql.sql
+    $ sudo systemctl start pdns
 
+or
 
-Now we have a database and an empty table. PowerDNS should now be able
-to launch in monitor mode and display no errors:
+.. code-block:: shell
 
-::
+    $ sudo systemctl restart pdns
 
-    # /usr/sbin/pdns_server --daemon=no --guardian=no --loglevel=9
-    (...)
-    15:31:30 PowerDNS 1.99.0 (Mar 12 2002, 15:00:28) starting up
-    15:31:30 About to create 3 backend threads
-    15:39:55 [gMySQLbackend] MySQL connection succeeded
-    15:39:55 [gMySQLbackend] MySQL connection succeeded
-    15:39:55 [gMySQLbackend] MySQL connection succeeded
+Make sure no error is reported, and use ``systemctl status pdns`` to make sure PowerDNS was started correctly.
 
-In a different shell, a sample query sent to the server should now
-return quickly *without* data:
+A sample query sent to the server should now return quickly *without* data:
 
 .. code-block:: shell
 
-    $ dig +short www.example.com @127.0.0.1  # should print nothing
-
-.. warning::
-  When debugging DNS problems, don't use ``host``. Please use
-  ``dig`` or ``drill``.
-
-And indeed, the output in the first terminal now shows:
+     dig a www.example.com @127.0.0.1 
 
-::
+    ; <<>> DiG 9.10.3-P4-Debian <<>> a www.example.com @127.0.0.1
+    ;; global options: +cmd
+    ;; Got answer:
+    ;; ->>HEADER<<- opcode: QUERY, status: REFUSED, id: 40870
+    ...
 
-    Mar 01 16:04:42 Remote 127.0.0.1 wants 'www.example.com|A', do = 0, bufsize = 1680: packetcache MISS
+.. warning::
+  When debugging DNS problems, don't use ``host``. Please use ``dig`` or ``drill``.
 
-Now we need to add some records to our database (in a separate shell):
+Note the ``REFUSED`` status - this is the code most name servers use to indicate they do not know about a domain.
 
-::
+Now, let's add a zone and some records:
```

Comment:
A paragraph introducing `pdnsutil`, and why raw SQL shouldn't be used would be nice here.